epsagon/aws_v2_wrapper.go

```diff
@@ -29,9 +38,9 @@ func getTimeStampFromRequest(r *aws.Request) float64 {
 	return float64(r.Time.UTC().UnixNano()) / float64(time.Millisecond) / float64(time.Nanosecond) / 1000.0
 }
 
-func completeEventData(r *aws.Request) {
-	defer GeneralEpsagonRecover("aws-sdk-go wrapper", "")
-	if tracer.GetGlobalTracerConfig().Debug {
+func completeEventData(r *aws.Request, currentTracer tracer.Tracer) {
+	defer GeneralEpsagonRecover("aws-sdk-go wrapper", "", currentTracer)
+	if currentTracer.GetConfig().Debug {
 		log.Printf("EPSAGON DEBUG OnComplete request response: %+v\n", r.HTTPResponse)
```

Comment:
not exactly possible. I mean, we can expose one more function in the Tracer interface but I am not sure we should at all (and I am not sure where to get the ID from + it might be generated when sending the trace)